Lombiq.HelpfulLibraries.OrchardCore/Workflow/DocumentedEventActivityDisplayDriverBase.cs

```diff
@@ -0,0 +1,83 @@
+﻿using Microsoft.AspNetCore.Mvc.Localization;
+using Microsoft.Extensions.Localization;
+using OrchardCore.DisplayManagement.Handlers;
+using OrchardCore.DisplayManagement.Notify;
+using OrchardCore.DisplayManagement.Views;
+using OrchardCore.Workflows.Activities;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Diagnostics.CodeAnalysis;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Lombiq.HelpfulLibraries.OrchardCore.Workflow;
+
+public class DocumentedEventActivityDisplayDriverBase<TActivity> : SimpleEventActivityDisplayDriverBase<TActivity>
+    where TActivity : class, IActivity
+{
+    protected readonly INotifier _notifier;
+    private readonly IStringLocalizer<DocumentedEventActivityDisplayDriver> T;
```

Comment:
Is the localizer intentionally not `IStringLocalizer<DocumentedEventActivityDisplayDriverBase> but IStringLocalizer<DocumentedEventActivityDisplayDriver>` ? Alternatively is the class `DocumentedEventActivityDisplayDriver` intentionally not derived from `DocumentedEventActivityDisplayDriverBase` ?
Consider these question as just questions first. If you consider this resolved feel free to close this.

Reply:
Yes. `IStringLocalizer<DocumentedEventActivityDisplayDriverBase>` is not possible, because that's a generic type so that would be `IStringLocalizer<DocumentedEventActivityDisplayDriverBase<TEvent>>`. My goal here is that if you want to localize you should only have to localize these texts once instead of once for every single event.
`DocumentedEventActivityDisplayDriver` only exists because I need a type that's not generic, it won't do anything besides being a marker for string localizer (it may contain other helper methods in the future, if necessary), so there is no need for it to inherit from `DocumentedEventActivityDisplayDriverBase`.